torch/ao/quantization/backend_config/README.md

```diff
@@ -22,7 +22,19 @@ Instead of hardcoding the fusion mappings, float to reference quantized module m
 
 ## Pattern Specification
 
-The operator patterns used in BackendConfig are float modules, functional operators and pytorch operators specified in reverse order:
+The operator patterns used in BackendConfig are float modules, functional operators, pytorch operators, or a tuple combination of the above. For example:
+* torch.nn.Linear
+* torch.nn.functional.linear
+* torch.add
+* operator.add
+* (torch.nn.functional.linear, torch.nn.functional.relu)
+* (torch.nn.Conv2d, torch.nn.BatchNorm2d, torch.nn.ReLU)
+
+Tuple patterns are treated as sequential patterns, and currently only tuples of 2 or 3 elements are supported.
+
+### Advanced Pattern Specification
+
+The above format should satisfy the vast majority of use cases. However, it does not handle more complex scenarios such as graph patterns. For these use cases, the BackendConfig API offers an alternative "reverse nested tuple" pattern format, enabled through `BackendPatternConfig()._set_pattern_complex_format(...)`. Note that this format is deprecated and will be replaced in a future version of PyTorch.
```

Comment:
should we name it something like `_set_pattern_complex_format_deprecated`?  If we are changing the name, might as well change to something we don't have to touch for awhile?

Reply:
what would be the name when we update the format to the subgraph rewriter format? do we want to reuse this or introduce a new name?

Reply:
I see, then the current name makes sense.  Maybe we can make `Note that this format is deprecated and will be replaced in a future version of PyTorch.` more specific and explain that we're not going to delete this format, the format spec will change in the future?

Reply:
yeah this matches what I was thinking before